magento/magento2
# Error on design configuration save with imageUploader form element po…

…pulated from gallery #21032

### Description (*)

This change fixes all file uploads on the theme config edit form page when is used "Select from Gallery" chooser of images.

### Fixed Issues (if relevant)

1. magento/magento2#21032: Error on design configuration save with imageUploader form element populated from gallery

### Manual testing scenarios (*)

1. Log in to Admin panel
2. Go to Content -> Design -> Configuration
3. Click "Edit" link in a grid
4. Expand "Header" fieldset
5. Click "Select from Gallery" button for " Logo Image" field
6. Select any image and click on "Add Selected" button
7. Click the "Save Configuration" button
8. The success message is shown (the image was stored without any errors)

### Contribution checklist (*)
